src/modules/elixir.rs

```diff
@@ -4,9 +4,15 @@ use crate::configs::elixir::ElixirConfig;
 use crate::formatter::StringFormatter;
 use crate::utils;
 
-use once_cell::sync::Lazy;
+use async_once::AsyncOnce;
+use lazy_static::lazy_static;
 use regex::Regex;
-use std::ops::Deref;
+
+lazy_static! {
+    static ref VERSIONS: AsyncOnce<Option<(String, String)>> =
+        AsyncOnce::new(async { get_elixir_version().await });
+}
```

Comment:
Doesn't `async` already have semantics that would make a suitable as a replacement to `Lazy` here? IIRC `async` functions don't get executed until they get polled. Or does `.await` imply taking ownership?